src/main/pages/che-6/user-guide/creating-starting-workspaces.adoc

```diff
@@ -10,41 +10,59 @@ folder: che-6/user-guide
+=== Duplicating an existing stack
 
-In this scenario, you need to create a stack first, and then use the resulting stack to create a workspace.
+Create a stack and then use the resulting stack to create a workspace.
 
 image::workspaces/duplicate_stack.gif[]
 
-*Create Custom stack from a custom link:recipes.html[recipe]*
+=== Creating a custom stack from a custom recipe
 
-In this scenario, you need to author a custom link:recipes.html[recipe], then a stack, and finally use the resulting stack to create a workspace.
+Author a custom link:recipes.html[recipe] and then create a stack. Use the resulting stack to create a workspace.
 
 image::workspaces/custom_recipe_stack.gif[]
 
 [id="starting-workspaces"]
-== Starting Workspaces
+== Starting workspaces
 
-A workspace can be started in one of the following ways:
+You can start a workspace in one of the following ways:
 
-1.  Auto-start when workspace is created in UD
-2.  Run or Open buttons in workspace details view in User Dashboard
-3.  By clicking on a workspace name in the left Dashboard sidebar (recent workspaces)
-4.  Using REST API. See link:rest-api.html[REST API] documentation
+* The workspace starts automatically after it is created in the user's *Dashboard*.
 
-It can take some time for a workspace environment to start. It depends on several factors like network conditions, Docker image availability, configured installers that may attempt to install additional tools and software in runtime etc. When a workspace is starting, you can track progress in a workspace start tab. There are also individual tabs for each machine in workspace environment that stream logs from installers (terminal, exec agent, ws agent, language servers if any).
+* In the user's *Dashboard*, use the *Run* or *Open* buttons in the *Workspace Details* view.
 
-[id="workspace-management"]
-== Workspace Management
+* Click a workspace name from the recent workspaces displayed in the left panel.
 
-Once a workspace is created or started, you can modify it by adding link:projects.html[projects], link:installers.html[installers], link:env-variables.html[env variables], link:volumes.html[volumes]. You can also edit raw workspace config, but beware that you may break your workspace this way, so always have a backup of a working config. If you change configuration of a running workspace and save it, a workspace will be restarted. Follow the links above to learn more about workspace configuration.
+* Use the REST API. For more information about the REST API, see link:rest-api.html[REST API].
+
+The workspace may take time to start depending on factors like network conditions, Docker image availability, configured installers attempting to install additional tools and software in the runtime. Track the progress of the workspace start operation in the *Workspace Start* tab. The tabs for each machine in the workspace environment, stream logs from the installers (terminal, exec agent, ws agent, language servers if any).
```

Comment:
Should "Docker image" be "container image"?

The last item in the list ("configured installers...") should have a conjunction.

No comma is needed in this sentence: "The tabs for each machine in the workspace environment stream logs from the installers (...)."

Should "exec" and "ws" be stylized?

Reply:
Also, conjunction before 'language servers'